README.adoc

```diff
@@ -204,24 +209,22 @@ In testing the functionality of the application, the scopes and dependencies are
 
 [role="code_command hotspot", subs="quotes"]
 ----
-#Create the `InventoryEndpointTest` class.#
-`src/test/java/it/io/openliberty/guides/inventory/InventoryEndpointTest.java`
+#Create the `InventoryEndpointIT` class.#
+`src/test/java/it/io/openliberty/guides/inventory/InventoryEndpointIT.java`
 ----
 
-The [hotspot=BeforeClass file=0]`@BeforeClass` annotation is placed on a method that runs before any of the test cases.
+The [hotspot=BeforeAll file=0]`@BeforeAll` annotation is placed on a method that runs before any of the test cases.
 In this case, the [hotspot=oneTimeSetup file=0]`oneTimeSetup()` method retrieves the port number for the Open Liberty server and builds
 a base URL string that is used throughout the tests.
 
-The [hotspot=Before file=0]`@Before` and [hotspot=After file=0]`@After` annotations are placed on methods that run before and after every test case.
+The [hotspot=BeforeEach file=0]`@BeforeEach` and [hotspot=AfterEach file=0]`@AfterEach` annotations are placed on methods that run before and after every test case.
```

Comment:
Should this be:
"The [hotspot=BeforeEach file=0]`@BeforeEach` and [hotspot=AfterEach file=0]`@AfterEach` annotations are placed on methods that run before **or** after every test case."?

It sounds odd to me with "and", but I could just be understanding it incorrectly.